From 93b9885a65e693d264e9199b42cf9a08b167b332 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 18 Aug 2015 11:53:31 +0200
Subject: [PATCH] network/ifup.sh: let dhcp client run in the background

we need to renew the lease

(cherry picked from commit fb2d643a385f1b02fbac6c2782b0026029c6bde7)
---
 modules.d/40network/ifup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/40network/ifup.sh b/modules.d/40network/ifup.sh
index bb20afe..ceb5f5c 100755
--- a/modules.d/40network/ifup.sh
+++ b/modules.d/40network/ifup.sh
@@ -109,7 +109,7 @@ do_dhcp() {
         info "Starting dhcp for interface $netif"
         dhclient "$@" \
                  ${_timeout:+-timeout $_timeout} \
-                 -1 -q \
+                 -q \
                  -cf /etc/dhclient.conf \
                  -pf /tmp/dhclient.$netif.pid \
                  -lf /tmp/dhclient.$netif.lease \
